spec/system/admin_category_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'AdminCategory crud', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:admin_user) { create(:admin_user) }
+  let(:category) { build(:category) }
+  let!(:reward) { create(:reward) }
+
+  it 'test admin crud actions' do
```

Comment:
I can see, that it is not in the AC, but I would consider adding specs for:
- editing category
- deleting category that has rewards
- deleting category without rewards

Reply:
Added to test.